test/parallel/test-module-readonly.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const path = require('path');
+const cp = require('child_process');
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+if (process.platform == 'win32') {
+    // Create readOnlyMod.js and set to read only
+    const readOnlyMod = path.join(tmpdir.path, 'readOnlyMod');
+    const readOnlyModRelative = path.relative(__dirname, readOnlyMod);
+    const readOnlyModFullPath = readOnlyMod + ".js";
+    fs.writeFileSync(readOnlyModFullPath, "module.exports = 42;");
+
+    // Removed any inherited ACEs, and any explicitly granted ACEs for the current user
+    cp.execSync(`icacls.exe "${readOnlyModFullPath}" /inheritance:r /remove "%USERNAME%"`);
```

Comment:
Long line here, whitespace/indent errors elsewhere.  Can you run `make lint`?

Reply:
As this is a Windows test the equivalent is `vcbuild lint` (or `vcbuild lint-js` to just lint the JavaScript files).